cmd/tf-operator.v1/app/server.go

```diff
@@ -119,6 +128,7 @@ func Run(opt *options.ServerOption) error {
 
 	// Set leader election start function.
 	run := func(<-chan struct{}) {
+		isLeader.Set(1)
```

Comment:
Should we just set this metric once from the pod or keep sending this repeatedly at an interval?

Reply:
Since a pod lose the leader status, shouldn't it be periodic?

Reply:
The catch here is that even if it is periodic, the value for the gauge is still set as 1. The periodicity might only signify freshness of the metric which the metric `up` is already providing us with (the success of scraping the /metric endpoint).
Should this metric only signify a pod's leadership?
How certain are we that `isLeader.Set(0)` is always hit on crash? If its not certain, how can we reset this metric to 0 when it is not the leader?